Description: Remove inline-related compiler warnings (by not using inline)
Author: B. Watson <yahlcru@gmail.com>
Last-Update: 2020-11-12

diff -Naur lv-4.51.orig/src/file.h lv-4.51.orig.patched/src/file.h
--- lv-4.51.orig/src/file.h	2020-11-12 01:41:53.353120244 -0500
+++ lv-4.51.orig.patched/src/file.h	2020-11-12 01:44:33.701105667 -0500
@@ -169,8 +169,8 @@
 # endif
 # define IobufFeof( a )		feof( (a)->iop )
 #else
-public inline int IobufGetc( iobuf_t *iobuf );
-public inline int IobufUngetc( int ch, iobuf_t *iobuf );
+public int IobufGetc( iobuf_t *iobuf );
+public int IobufUngetc( int ch, iobuf_t *iobuf );
 public offset_t IobufFtell( iobuf_t *iobuf );
 public int IobufFseek( iobuf_t *iobuf, offset_t off, int mode );
 public int IobufFeof( iobuf_t *iobuf );
